test/README.md

```diff
@@ -1,8 +1,34 @@
 ![PODMAN logo](../logo/podman-logo-source.svg)
-# Integration Tests
+# Test utils
+Test utils provide common functions and structs for testing. It includes two structs:
+* `PodmanTest`: Handle the *podman* command and other global resources like temporary
+direcotry. It privides basic methods, like checking podman image and pod status. Test
```

Comment:
Might want to run `aspell` on this: "direcotry", "privides" are misspelled. There's also a `sturct` somewhere below.

Reply:
Thanks a lot. Will check the doc with aspell and update it.